viewer/apiSessions.js

```diff
@@ -3013,7 +3013,7 @@ class SessionAPIs {
       console.log('/api/session/entire/%s/%s/pcap query', ArkimeUtil.sanitizeStr(req.params.nodeName), ArkimeUtil.sanitizeStr(req.params.id), JSON.stringify(query, false, 2));
     }
 
-    Db.searchSessions(['sessions2-*', 'sessions3-*'], query, null, (err, data) => {
+    Db.searchSessions(Db.defaultIndexPatterns(Config.getArray('queryExtraIndices', '')), query, null, (err, data) => {
```

Comment:
Throughout the code the several places where `['sessions2-*', 'sessions3-*']` are passed into db.js have been replaced by `Db.defaultIndexPatterns` which takes an `extraIndices` array (pulled from the config's `queryExtraIndices` value) and merges its contents with the default `['sessions2-*', 'sessions3-*']` array. This merged array is returned.